### ISSUE TITLE
Loss does not converge during training, which results in a poor evaluation results(map:41.92%)

### ISSUE THREAD

**Issue by Sebastian514** [hunglc007/tensorflow-yolov4-tflite#394]
I used the default hyper-parameters and trained the model on a single Nvidia Titan Xp.  And the training logs show that the loss   
didn't converge. It apparently leaded to a bad performance which evaluated on coco.

What should I do?

**Comment by DUANlaoda** [comment]
你好，你解决 loss 是 nan的问题了吗？
